From: SeongJae Park <sj@kernel.org>
Subject: mm/damon: remove damon_ctx->callback kernel-doc comment
Date: Tue, 15 Jul 2025 11:52:39 -0700

Commit 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback")
on mm-unstable tree removed callback field of damon_ctx, but not erased
the kernel-doc comment for the field.  This results in build warning.  Fix
it by removing the obsolete comment.

Link: https://lkml.kernel.org/r/20250715185239.89152-3-sj@kernel.org
Signed-off-by: SeongJae Park <sj@kernel.org>
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/damon.h |    1 -
 1 file changed, 1 deletion(-)

--- a/include/linux/damon.h~mm-damon-core-add-cleanup_target-ops-callback-fix-2
+++ a/include/linux/damon.h
@@ -774,7 +774,6 @@ struct damon_attrs {
  * Accesses to other fields must be protected by themselves.
  *
  * @ops:	Set of monitoring operations for given use cases.
- * @callback:	Set of callbacks for monitoring events notifications.
  *
  * @adaptive_targets:	Head of monitoring targets (&damon_target) list.
  * @schemes:		Head of schemes (&damos) list.
_
